parquet-thrift/src/test/java/org/apache/parquet/hadoop/thrift/TestBinary.java

```diff
@@ -81,9 +80,13 @@ public void testBinary() throws IOException {
   private void assertSchema(ParquetMetadata parquetMetadata) {
     List<Type> fields = parquetMetadata.getFileMetaData().getSchema().getFields();
     assertEquals(2, fields.size());
-    assertEquals(Types.required(PrimitiveType.PrimitiveTypeName.BINARY).named("s"), fields.get(0));
```

Comment:
Why not add the ID to this type instead of verifying each field individually?

Reply:
I recommended Nandor to check the fields separately after I asked him what he would like to assert. Since he wanted to check name, primary type and logical type, but the equals method checks the name, primary type and ID (so the logical type is not checked, but the ID is), I thought it would be cleaner to assert on the actual fields we care about.

If you prefer, we can also do an equals check on the object (with an ID added to the reference value) and an additional separate check for the logical type field.

Reply:
I don't see a reason not to use normal type equality for this check.

Reply:
Done, added IDs for the expected type.